src/game_state/player.rs

```diff
@@ -72,186 +49,172 @@ pub struct ReportMessage {
     pub conn_id: Uuid,
 }
 
-#[derive(Debug)]
-pub struct PlayerMovementMessage {
-    pub conn_id: Uuid,
-    pub new_position: Position,
-}
-
 #[derive(Debug)]
 pub struct PlayerMoveAndLookMessage {
```

Comment:
I've decided to just have one move and look message rather than move, look, and move and look. I think leveraging the options here works just fine. It reduces boilerplate and in this context it makes sense that a players 'movement' might include the angle they are facing as well.